Tests/Fixtures/TestProject/fixtures.xcconfig

```diff
@@ -0,0 +1,7 @@
+#include "xcode12_workaround.xcconfig"
+
+// Common settings for fixtures
+CODE_SIGN_IDENTITY =
+CODE_SIGNING_REQUIRED = NO
+CODE_SIGN_ENTITLEMENTS =
+CODE_SIGNING_ALLOWED = NO
```

Comment:
Anything not related to Xcode 12, can we put in the actual spec? I'd like to have that as buildable as possible without custom xcodebuild args